.gitignore

```diff
@@ -10,3 +10,13 @@ sdk/node_modules
 .vscode/
 **/.yalc/
 **/yalc.lock
+dist
+artifacts
```

Comment:
`artifacts` are now included in the build step of the legacy sdk. This way it will always have the latest version and we don't have to manually copy them over. Because of them being automatically copied over during build we don't have to include in git anymore

Reply:
it makes sense.
Is there an edge-case that 1) clone repo 2) edit contract 3) anchor test  (no artifact?) .
If so how about adding step to instllation process to build SDK package at least once.

Reply:
If you run 'yarn test' instead of 'anchor test' it resolves and builds all dependencies correctly

Reply:
> yarn test
noted! 🫡